docs/docs/06_plugins/01_available-plugins/01-transactional/04-pg-promise-adapter.md

```diff
@@ -52,6 +57,8 @@ ClsModule.forRoot({
 
 The `tx` property on the `TransactionHost<TransactionalAdapterPgPromise>` is typed as [`Database`](https://vitaly-t.github.io/pg-promise/Database.html).
 
+`@Transactional()` takes an optional argument that is the same as the `option` argument to `Database.tx()` in `pg-promise` ([docs](https://vitaly-t.github.io/pg-promise/Database.html#tx)).
+
```

Comment:
Not sure this specification is needed, it is already established in [the general docs](https://papooch.github.io/nestjs-cls/plugins/available-plugins/transactional#passing-transaction-options).